ui/packages/tidb-dashboard-for-clinic/.env.production

```diff
@@ -0,0 +1,5 @@
+REACT_APP_VERSION=$npm_package_version
+REACT_APP_MIXPANEL_HOST=https://telemetry.pingcap.com/api/v1/dashboard/report
+# TODO: Move to GitHub Secrets
+# DO NOT COPY THE TOKEN TO OTHER PROJECTS
+REACT_APP_MIXPANEL_TOKEN=
```

Comment:
How do we distinguish tokens of different projects in a monolithic dashboard repository?

Reply:
oh, I missed its value.